lib/govuk_tech_docs/api_reference/templates/api_reference_full.html.erb

```diff
@@ -1,7 +1,8 @@
 <h1 id="<%=  info.title.parameterize %>"><%= info.title %> v<%= info.version %></h1>
 <%= info.description_html %>
 
-<% unless servers.empty? %>
+<%# OpenAPI files default to having a single server of URL "/" %>
```

Comment:
```suggestion
<% # OpenAPI files default to having a single server of URL "/" %>
```

Reply:
`<%#` is actually erb syntax for a comment and means something different to `<% #`: https://docs.ruby-lang.org/en/2.3.0/ERB.html#class-ERB-label-Recognized+Tags

Reply:
Ooh, TIL! 👀